woocommerce/wc-admin
# Update packages with vulnerabilities

Updates some packages with vulnerabilities that had new upstream versions were available.

Before:
`found 70 vulnerabilities (67 low, 3 moderate)`

After:
`found 2 vulnerabilities (1 low, 1 moderate)`

### Detailed test instructions:
- Remove installed packages: `rm -rf node_modules`.
- Install packages again: `npm i`.
- Verify NPM reports less vulnerabilities than before.
- Run `npm start`.
- Interact with the app and verify there are no regressions.
